pacwrapper.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"bytes"
+	"log"
+	"net/http"
+	"text/template"
+)
+
+type PACData struct {
+	Port     int
+	PACURL   string
+	Domain   string
+	Username string
```

Comment:
What does this use the Domain and Username for? I couldn't find any references to it elsewhere in the code...

Reply:
...also I couldn't figure out what PACURL is used for either...

Reply:
I had in mind the next patch where a user would be able to supply their own wrapper, so I added all the context available to the struct so it would be available to the template. I can remove it if you like. Or should I write that next patch?

Reply:
If it's part of another patch, I do feel like it should go together with that other patch.

Reply:
Ok. Done in e928fe0a4591e0ed00b54e5e6f97fc807de6dda7